src/battle-text-parser.ts

```diff
@@ -136,7 +136,7 @@ class BattleTextParser {
 				return {args: ['-fieldactivate', effect], kwArgs: {}};
 			}
 
-			if (id === 'symbiosis' || id === 'poltergeist') {
+			if (id === 'symbiosis') {
```

Comment:
You might want to avoid breaking old replays. This would also mean that the client change can be merged first, and then the server change later.

```suggestion
			if (id === 'poltergeist') {
				return {args: ['-item', arg3], kwArgs: {from: effect}};
			}

			if (id === 'symbiosis') {
```

Reply:
In general, we always want to avoid breaking replays unless there's a good reason otherwise.